Homework/Develop/assets/css/style.css

```diff
@@ -8,39 +8,43 @@ body {
     background-color: #d9dcd6;
```

Comment:
Maybe worth investigating if we can add a `font-family` here if it being used across the web page and for cases when they are different we can overwrite them in the respective styles.

Reply:
updated the body tag to include the font family and removed the repeated ones in other tags here dd90339a4aa6736e8ec16f222bd65186e4b010a9